doc/source/index.rst

```diff
@@ -1,4 +1,4 @@
-PyAnsys Units documentation |version|
+PyAnsys units documentation |version|
```

Comment:
@hpohekar now that you added "PyAnsys Units" to accept.txt, we should try to remain consistent throughout. Are we allowed to write Units here now without triggering errors.

Reply:
I have already updated it you are referring to an old change.

Reply:
Are we allowed to write Units here now without triggering errors? - still error is shown by vale.

Reply:
I see. I believe that we should be consistent then with what we did for other libraries - see the following example. PyAnsys units doesn't look good somehow (the units part is not carrying enough weight).

![image](https://github.com/ansys/pyansys-units/assets/93727996/648f7532-cafd-4c58-ba7b-b62fa0dc63de)